functional/testHeadlessComponents/README.md

```diff
@@ -0,0 +1,26 @@
+# External TestHeadlessComponents Tests
+
+Test of correct behavior of swing components in a headless jre environment.
+
+## Running TestHeadlessComponents locally
+
+### Setting up the environment:
+Nothing special is required, just install the java package or unpack the portable you want the test.
+
+### ENV variables:
+The test utilizes *JAVA_HOME variable, so if it is not set it will not work correctly! In case of testing portable packages please set it up first.
+BOOTJDK_DIR - directory to store bootjdk which is used to compile the code in case we are testing jre only installation. Uses ~/bootjdk by default and uses Adoptium latest build of relevant jdk for this purpose.
+BOOTJDK_ARCHIVE_DIR - in case the user wants to use arbitrary jdk build, he can provide path to its archive and it will be used in jre execution. Creates $WORKSPACE/bootjdkarchive and downloads latest temurin archive if not set.
+WORKSPACE - directory where the testsuite is going to execute all the tests. ~/workspace by default
+TMPRESULTS - this is a location where the logfiles will be after the testsuite finishes. Same as WORKSPACE by default.
+*JREJDK - This tells the testsuite whether we are testing jre or jdk.
```

Comment:
AQAvit has a USE_JRE parameter that could be leveraged, but won't block this PR regarding its use.

Reply:
oh talking about parameters.. is there anything saying whether the testmachine has a display output? we could use it to enable a few more testscases via XDISPLAY variable.

Reply:
We set XDISPLAY via the Xvfb Jenkins plugin in our test pipeline code here: https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/JenkinsfileBase#L735-L757 (exceptions for AIX and solaris platforms, where Xvfb plugin does not work, those are set without the plugin as shown in that code snippet).

So, machines at this Jenkins server (ci.adoptium.net) have capability for XDISPLAY.  

I should note that for a couple of our Temurin distributions, we **only** produce a headless build (alpine-linux and riscv).  I presume these testcases are still valid against them, but will double-check via Grinders that they run as expected.

Reply:
and how does the USE_JRE parameter work(what are the possible values)? I need to have JREJDK filled either with a value of "jre" or "jdk" so it knows what kind of behavior it can expect.

Reply:
USE_JRE is an environment variable, that if set to true, means we TEST_JDK_HOME (the binary that we want to test) to the JRE (rather than the JDK) so that particular tests are run with the JRE rather than the JDK that we pick up 

I am happy to bring this PR in as it stands and come back with a follow-up as needed (once we think this through more thoroughly... my worry is whether some vendors do not produce JREs for later versions of Java, you'll still will want to check at the test level whether its available or not).